c10/macros/Export.h

```diff
@@ -144,6 +144,12 @@
 #define TORCH_HIP_API C10_IMPORT
 #endif
 
+#if defined(TORCH_XPU_BUILD_MAIN_LIB)
+#define TORCH_XPU_API C10_EXPORT
+#else
+#define TORCH_XPU_API C10_IMPORT
+#endif
+
```

Comment:
Why this is needed, if this PR does not introduce `TORCH_XPU_API`?

Reply:
Sorry, `TORCH_XPU_API` is used in `aten`. I will remove this API, and add it back when needed.